tests/unit-tests/blocks/test-class-sensei-continue-course-block.php

```diff
@@ -64,9 +64,9 @@ public function testRender_NotEnrolled_ReturnsEmptyString() {
 
 	/**
 	 * Doesn't render the block if the student is enrolled and has already completed the course.
-	 *
-	 * @covers Sensei_Continue_Course_Block::render
-	 */
```

Comment:
ouch, not good :)

Reply:
@merkushin fixed here [45d05ee](https://github.com/Automattic/sensei/pull/5591/commits/45d05ee7679a27dd471795e0a6d8caa9516e2626)